_std/defines/speech_defines/modules.dm

```diff
@@ -31,6 +31,7 @@
 #define SPEECH_OUTPUT_THRALLCHAT_GLOBAL "global_thrallchat"
 #define SPEECH_OUTPUT_THRALLCHAT_THRALL "thrall_thrallchat"
 #define SPEECH_OUTPUT_THRALLCHAT_VAMPIRE "vampire_thrallchat"
+#define SPEECH_OUTPUT_MENTOR_MOUSE "mentor_mouse"
```

Comment:
All speech defines should be in alphabetical order please.

Reply:
added